components/Match/matchColumns.jsx

```diff
@@ -249,42 +249,109 @@ const laningColumns = [
     field: 'lane_efficiency',
     width: 1,
     sortFn: true,
+    displayFn: (row, col, field) => field.toFixed(2),
   }, {
     displayName: 'LH@10',
     field: 'lh_t',
     width: 1,
     sortFn: true,
+    displayFn: (row, col, field) => field ? field[10] : '',
   }, {
     displayName: 'DN@10',
     field: 'dn_t',
     width: 1,
     sortFn: true,
+    displayFn: (row, col, field) => field ? field[10] : '',
   }];
 
+const purchaseColumns = [
+  heroTdColumn,
+  {
+    displayName: 'TP',
+    tooltip: 'Town Portal Scrolls purchased',
+    field: 'purchase',
+    width: 1,
+    sortFn: true,
+    displayFn: (row, col, field) => field ? field.tpscroll : '',
```

Comment:
I think another way you could do this (without using ternary everywhere) is 
```js
(row, col, { tpscroll }) => tpscroll
// or maybe
(row, col, { tpscroll = '' }) => tpscroll
```
I'm not sure if the default value is needed or not. Not sure if it's better or worse, but something to think about.